xwiki-platform-core/xwiki-platform-realtime/xwiki-platform-realtime-wysiwyg/xwiki-platform-realtime-wysiwyg-test/xwiki-platform-realtime-wysiwyg-test-docker/src/test/it/org/xwiki/realtime/wysiwyg/test/ui/RealtimeWYSIWYGEditorIT.java

```diff
@@ -761,7 +761,7 @@ void editSameMacro(TestReference testReference, TestUtils setup)
         firstMacroEditModal.clickCancel();
 
         firstEditPage.clickSaveAndView();
-        assertEquals("{{info cssClass=\"bar\" title=\"Some cool title\"}}\ntwo one\n{{/info}}\n\n ",
+        assertEquals("{{info cssClass=\"bar\" title=\"(% class=~\"box-title box-title~\" %)Some cool title\"}}\ntwo one\n{{/info}}\n\n ",
```

Comment:
Why do we get this strange wiki syntax? This seems wrong.

Reply:
I fixed it [in rendering](https://github.com/xwiki/xwiki-rendering/pull/296/commits/d64771c76e26747f0b529b43f7257a2de447873b). Somehow the FormatBlock was interacting weirdly with its child Metadata when editing inline. The GroupBlock doesn't have this issue and behaves nicely. This also fixes the extra newlines that were added in the parsing of the block content.
WIP: reverting the changes on most tests and making sure they still pass successfully